yb-voyager/cmd/importSchema.go

```diff
@@ -86,24 +86,42 @@ func importSchema() error {
 
 	tconf.Schema = strings.ToLower(tconf.Schema)
 
+	tconfSchema := tconf.Schema
```

Comment:
A little concerned about this change. We don't usualyl use tdb in import-schema. Even if you look at line 90, we are doing some workarounds, just to get tdb working somehow for callhome. Let me think about this and get back. cc: @priyanshi-yb

Reply:
+1

Reply:
Instead of introducing tdb here, how about just using raw connections directly (like how the rest of import-schema is implemented) to figure out if you have the right permissions.

Not defining `GetMissingImportSchemaPermissions` in target db interface also makes sense because we only import schema into YB now.